README.md

```diff
@@ -242,6 +246,19 @@ CREATE TABLE checks (name TEXT NOT NULL PRIMARY KEY, timestamp INT8, results TEX
 
 Currently the status page does not support SQL storage.
 
+#### Slack notifier
+
+slack configuration:
```

Comment:
"Enable notifications in Slack with this Notifier configuration:"

Reply:
done